index.html

```diff
@@ -214,5 +214,45 @@ <h3>Multi-Post Stories</h3>
 		  </ul>
 		</section>
 	</main>
+
+	<!-- The footer with the form and social links -->
+	<footer class="footer">
+		<div class="ttl-para">
+			<h2 class="contact-us">contact us</h2>
+			<p class="para ftr-para">
+				If you have an application you are interested in developing, 
+				a feature that you need built or a project that needs coding. I’d love to help with it 
+			</p>
+		</div>
+
+		<!-- The form -->
+		<form  action="https://formspree.io/f/mnqlvaya"	method="POST">
+			<div>
+				<label for="name">Name</label>
+				<input type="text" name="name" class="name name-email" id="name" maxlength="30" required placeholder="Enter your name">
+	
+				<label for="email">Email</label>
+				<input type="email" name="email" class="email name-email" id="email" required placeholder="email@email.com">
+			</div>
+
+			<textarea name="message" class="message" maxlength="500" placeholder="Enter text here."></textarea>
+
+			<button type="submit" class="see-project">
```

Comment:
- [ ] With the original webpage(index file), on submitting the form, the contents of form fields are not submitted successfully. Give this a fix

Reply:
done, thanks!